modules/rosa-sts-creating-a-cluster-using-defaults-ocm.adoc

```diff
@@ -61,11 +61,4 @@ If your AWS account ID is not listed, check that you have successfully associate
 [NOTE]
 ====
 If the installation fails or the cluster *State* does not change to *Ready* after about 40 minutes, check the installation troubleshooting documentation for details. For more information, see _Troubleshooting installations_. For steps to contact Red Hat Support for assistance, see _Getting support for Red Hat OpenShift Service on AWS_.
-====
-
-ifeval::["{context}" == "rosa-sts-creating-a-cluster-quickly"]
-:quick-install:
-endif::[]
-ifeval::["{context}" == "rosa-quickstart"]
-:quickstart:
-endif::[]
```

Comment:
The opening of these conditions still exists in lines 10-15 above. Pretty sure this will cause build issues?